.github/workflows/gh-ci.yaml

```diff
@@ -15,165 +15,170 @@ defaults:
 
 env:
   MDA_CONDA_MIN_DEPS: "pip pytest==6.1.2 mmtf-python biopython networkx cython matplotlib-base scipy griddataformats hypothesis gsd codecov"
-  MDA_CONDA_EXTRA_DEPS: "seaborn>=0.7.0 clustalw=2.1 netcdf4 scikit-learn joblib>=0.12 chemfiles tqdm>=4.43.0 tidynamics>=1.0.0 rdkit>=2020.03.1 h5py==2.10.0"
-  MDA_PIP_MIN_DEPS: 'coveralls coverage<5 pytest-cov pytest-xdist'
-  MDA_PIP_EXTRA_DEPS: 'duecredit parmed'
-
+  MDA_CONDA_EXTRA_DEPS: "seaborn>=0.7.0 netcdf4 scikit-learn joblib>=0.12 chemfiles>=0.9 tqdm>=4.43.0 tidynamics>=1.0.0 rdkit>=2020.03.1 h5py==2.10.0"
+  MDA_PIP_MIN_DEPS: "coveralls coverage<5 pytest-cov pytest-xdist"
+  MDA_PIP_EXTRA_DEPS: "duecredit parmed"
+  # Install clustalw separately since it needs a specific conda channel (biobuilds)
+  # which do not play nicely with conda-forge
+  MDA_CLUSTALW_DEP: "clustalw=2.1"
 
 jobs:
   main_tests:
     if: "github.repository == 'MDAnalysis/mdanalysis'"
     runs-on: ${{ matrix.os }}
     strategy:
-        fail-fast: false
-        matrix:
-          os: [ubuntu-latest, ]
-          python-version: [3.6, 3.7, 3.8]
-          run_type: [FULL, ]
-          install_hole: [true, ]
-          codecov: [true, ]
-          include:
-            - name: macOS
-              os: macOS-latest
-              python-version: 3.7
-              run_type: FULL
-              install_hole: true
-              codecov: true
-            - name: minimal-ubuntu
-              os: ubuntu-latest
-              python-version: 3.6
-              run_type: MIN
-              install_hole: false
-              codecov: true
-            - name: numpy_min
-              os: ubuntu-latest
-              python-version: 3.6
-              run_type: FULL
-              install_hole: false
-              codecov: false
-              numpy: numpy=1.16.0
-            - name: asv_check
-              os: ubuntu-latest
-              python-version: 3.7
-              run_type: FULL
-              install_hole: false
-              codecov: false
+      fail-fast: false
```

Comment:
@Luthaf can you un-shift all these? It makes it impossible to see where the changes have been introduced.

Reply:
sorry, auto-formatter =/ I'll undo these!

Reply:
Sorry... can I ask you to re-undo the auto-formatter again? I realise we should have shifted everything by 4 originally, but at this point it's going to make git blame difficult :(

Reply:
Really sorry about that. I'll check how to disable the formatter for mdanalysis code!